tests/Integration/Subscriber/CDN/RocketCDN/NoticesSubscriber/displayRocketcdnCta.php

```diff
@@ -18,9 +18,52 @@ private function getActualHtml() {
 		return $this->format_the_html( ob_get_clean() );
 	}
 
-	/**
-	 * Test should not display notice when RocketCDN is active
-	 */
+	public function testShouldDisplayNothingWhenNotLiveSite() {
+		$callback = function() {
+			return 'http://localhost';
+		};
+	
+		$not_expected = $this->format_the_html( '<div class="wpr-rocketcdn-cta-small notice-alt notice-warning wpr-isHidden" id="wpr-rocketcdn-cta-small">
+			<div class="wpr-flex">
+				<section>
+					<h3 class="notice-title">Speed up your website with RocketCDN, WP Rocket’s Content Delivery Network.</strong></h3>
+				</section>
+				<div>
+					<button class="wpr-button" id="wpr-rocketcdn-open-cta">Learn More</button>
+				</div>
+			</div>
+		</div>
+		<div class="wpr-rocketcdn-cta " id="wpr-rocketcdn-cta">
+			<section class="wpr-rocketcdn-cta-content--no-promo">
+				<h3 class="wpr-title2">RocketCDN</h3>
+				<p class="wpr-rocketcdn-cta-subtitle">Speed up your website thanks to:</p>
+				<div class="wpr-flex">
+					<ul class="wpr-rocketcdn-features">
+						<li class="wpr-rocketcdn-feature wpr-rocketcdn-bandwidth">High performance Content Delivery Network (CDN) with <strong>unlimited bandwith</strong></li>
+						<li class="wpr-rocketcdn-feature wpr-rocketcdn-configuration">Easy configuration: the <strong>best CDN settings</strong> are automatically applied</li>
+						<li class="wpr-rocketcdn-feature wpr-rocketcdn-automatic">WP Rocket integration: the CDN option is <strong>automatically configured</strong> in our plugin</li>
+					</ul>
+					<div class="wpr-rocketcdn-pricing">
+						<h4 class="wpr-rocketcdn-pricing-current"><span class="wpr-title1">$7.99</span> / month</h4>
+						<button class="wpr-button wpr-rocketcdn-open" data-micromodal-trigger="wpr-rocketcdn-modal">Get Started</button>
+					</div>
+				</div>
+			</section>
+			<div class="wpr-rocketcdn-cta-footer">
+				<a href="https://go.wp-rocket.me/rocket-cdn" target="_blank" rel="noopener noreferrer">Learn more about RocketCDN</a>
+			</div>
+			<button class="wpr-rocketcdn-cta-close--no-promo" id="wpr-rocketcdn-close-cta">
+				<span class="screen-reader-text">Reduce this banner</span>
+			</button>
+		</div>' );
```

Comment:
Shouldn't this html live in a fixture file?

Reply:
If it's reusable for other tests, then yes. A fixture gives us the ability to pull in a fixture for multiple tests. 

If no, then it's okay to have it directly in the test itself. Why? One advantage is: you can quickly see the HTML that should be generated, i.e. the expected behavior.